source/agora/node/Registry.d

```diff
@@ -195,29 +195,17 @@ public class NameRegistry: NameRegistryAPI
     }
 
     /// Internal endpoint, mimics `getValidator` but forward the query if needed
-    public final const(RegistryPayloadData) getValidatorInternal (in PublicKey key)
+    public final void getValidatorInternal (in PublicKey key,
```

Comment:
Let's return bool here,, so that we dont need the `retVal` trick in the caller site?

Reply:
Not needed since #3053 removes `retry`